### DISCUSSION TITLE
Working of older version of moose

### DISCUSSION THREAD

**Discussion by K-Komal-98** [idaholab/moose#22179] [Q&A Getting Started] [answered]
I am running a code of 2018 but that is not converging now. So, I want to run an older version of moose of april 2018 to reproduce the results. I followed these steps in order to install the older version:
conda create -n test moose-mpich
conda activate test
cd projects
cd moose
git checkout e1abc87f1b8ca4a7b8ba0b2fd9f803fb2ee31c8     #moose version

I checked the moose folder and saw that the update_and_rebuild_petsc.sh and many other files are missing. Where can I have gone wrong?
On reading the documentation and the forum, 2 more commands has to be run
./scripts/update_and_rebuild_petsc.sh
./scripts/update_and_rebuild_libmesh.sh
but this cannot be run unless I have the update_and_rebuild_petsc.sh file.

**Comment by GiudGiud** [comment, marked as answer]
Hello

In 2018 we probably did not have these scripts. That was 4 years ago. In fact the petsc script is from December 2018
You ll have to take these steps manually
OR
you can pull newer versions of the scripts. libmesh and petsc should be fairly backwards compatible

Guillaume
